src/systems/filecoin_vm/sysactors/reward_actor.id

```diff
@@ -0,0 +1,30 @@
+import vmr "github.com/filecoin-project/specs/systems/filecoin_vm/runtime"
+import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
+import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
+import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
+
+type Reward struct {
+    StartEpoch      block.ChainEpoch
+    Value           actor.TokenAmount
+    ReleaseRate     actor.TokenAmount  // linear slope, unit: FIL per Epoch
+    WithdrawAmount  actor.TokenAmount
```

Comment:
[Blocking] rename to `AmountWithdrawn` (or something similar) to avoid confusion